Krypton

deltha opened the pull request:
Hello,

!! Important, would you please take a look before the approval, maybe I did something wrong merge 111 commits into master? !!
In the archive there are 73 new files.


### Description
<!--- Provide a short summary of submitted add-on in case it's a new addition. -->
<!--- If it's plugin update only highlight biggest changes if needed. -->
<!--- Make sure you follow the checklist below before finalizing your pull-request. -->
### Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply like this: [X] -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

 - Add-on name: TehniK.ro
 - Add-on ID: plugin.video.tehnik
 - Version number: 2.0.7
 - Kodi/repository version: Krypton 17.x 
 - Code location URL: https://github.com/deltha/plugin.video.tehnik
 - Revision/tag: 769c5befb755d9f0590e9e361457215a9a5a4720
 - Branch: master

TehniK.ro Krypton Plugin Upgrade targeting the base branch for krypton
I attach the zip file downloaded from 
https://github.com/deltha/plugin.video.tehnik

What is new in this version

Menu IT: Removed Axel Tech - content changes.
Menu Mobiles: Added George Buhnici - content changes.
Games: Removed Tiranianu Gaming
Replaced few png files.

It was tested with Codacy

deltha-MBP-2:repo-plugins user$ git remote -v
myaddon	git@github.com:deltha/plugin.video.tehnik.git (fetch)
myaddon	git@github.com:deltha/plugin.video.tehnik.git (push)
origin	git@github.com:xbmc/repo-plugins.git (fetch)
origin	git@github.com:xbmc/repo-plugins.git (push)

What has been done:
http://kodi.wiki/view/HOW-TO:Create_add-on_PRs_using_Git_Subtree_Merging

deltha-MBP-2:GitHub user$ git clone git@github.com:xbmc/repo-plugins.git
deltha-MBP-2:GitHub user$ cd repo-plugins/
deltha-MBP-2:repo-plugins user$ git remote add myaddon git@github.com:deltha/plugin.video.tehnik.git
deltha-MBP-2:repo-plugins user$ git fetch myaddon
deltha-MBP-2:repo-plugins user$ git checkout -b myaddon_branch myaddon/master
deltha-MBP-2:repo-plugins user$ git checkout krypton
deltha-MBP-2:repo-plugins user$ git rm -r plugin.video.tehnik/
deltha-MBP-2:repo-plugins user$ git read-tree --prefix=plugin.video.tehnik/ -u myaddon_branch
deltha-MBP-2:repo-plugins user$ git diff --staged
deltha-MBP-2:repo-plugins user$ git commit -m "[plugin.video.tehnik] 2.0.7"
[krypton c2f93cb3] [plugin.video.tehnik] 2.0.7
 73 files changed, 924 insertions(+)

deltha-MBP-2:repo-plugins user$ git push origin krypton

ERROR: Permission to xbmc/repo-plugins.git denied to deltha.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.


Would you please be so kind to help me?

I have created a PR in the directory https://github.com/xbmc/repo-plugins/compare/krypton?expand=1
I attach in the PR the zip file downloaded from 
https://github.com/deltha/plugin.video.tehnik


Thank you in advance,
Remus Rujinschi
deltha@gmail.com
[plugin.video.tehnik.zip](https://github.com/xbmc/repo-plugins/files/924595/plugin.video.tehnik.zip)

Razzeee commented:
@deltha 
You should not try to merge our branches against each other.

1. You need to make your own fork (hit fork here https://github.com/xbmc/repo-plugins)
2. Push the changes to YOUR branch
3. Then do the PR here, referencing your branch as source and ours as base.

deltha commented:
Hi,

I have created a new one https://github.com/xbmc/repo-plugins/pull/1159
Referring to pull 1158 Thanks for the advices and for your patience.


I have done the following:

First step:
1. You need to make your own fork (hit fork here
https://github.com/xbmc/repo-plugins)

After I have logged in into github I was going to
https://github.com/xbmc/repo-plugins
and pressed the fork button to the upper right to fork a copy in my
repository which is https://github.com/deltha
and the fork has appeared in my github
repo-plugins
Forked from xbmc/repo-plugins
Then I have clonned it.


deltha-MBP-2:GitHub user$ git clone git@github.com:deltha/repo-plugins.git
Cloning into 'repo-plugins'...
remote: Counting objects: 61302, done.
remote: Compressing objects: 100% (92/92), done.
remote: Total 61302 (delta 10), reused 2 (delta 2), pack-reused 61202
Receiving objects: 100% (61302/61302), 404.90 MiB | 6.64 MiB/s, done.
Resolving deltas: 100% (22608/22608), done.
deltha-MBP-2:GitHub user$ cd repo-plugins/



I have added my git repository where your addon resides as a remote which
is : github.com:deltha/plugin.video.tehnik
So:

deltha-MBP-2:repo-plugins user$ git remote add myaddon git@github.com:
deltha/plugin.video.tehnik.git
deltha-MBP-2:repo-plugins user$ git fetch myaddon
warning: no common commits
remote: Counting objects: 80, done.
remote: Compressing objects: 100% (70/70), done.
remote: Total 80 (delta 8), reused 80 (delta 8), pack-reused 0
Unpacking objects: 100% (80/80), done.
From github.com:deltha/plugin.video.tehnik
 * [new branch]        master     -> myaddon/master


I have created a branch for my addon:


deltha-MBP-2:repo-plugins user$ git checkout -b myaddon_branch
myaddon/master
Branch myaddon_branch set up to track remote branch master from myaddon.
Switched to a new branch 'myaddon_branch'

This directory contains only the files from my addon, you can see if I list
the directory:

deltha-MBP-2:repo-plugins user$ ls
INSTALL.txt    LICENSE.txt    addon.py    addon.xml    changelog.txt
fanart.jpg    icon.png    resources
deltha-MBP-2:repo-plugins user$ head -8 addon.py
#!/usr/bin/python
#------------------------------------------------------------
# License: GPL (http://www.gnu.org/licenses/gpl-3.0.html)
# plugin.video.tehnik:
# By: deltha@gmail.com
#------------------------------------------------------------
# TehniK - 2.0.7
#------------------------------------------------------------


Now I am switching to the krypton branch of repo-plugins repo

deltha-MBP-2:repo-plugins user$ git checkout krypton
Branch krypton set up to track remote branch krypton from origin.
Switched to a new branch 'krypton'

deltha-MBP-2:repo-plugins user$ ls
README.md            plugin.video.cbcnews        plugin.video.ispot
plugin.video.phoenixmediathek    plugin.video.swrmediathek
info.txt            plugin.video.charge
plugin.video.kikamediathek    plugin.video.raiyoyo        plugin.video.tbdtv
plugin.audio.detektorfm        plugin.video.crossroads
plugin.video.mdrmediathek    plugin.video.revision3        plugin.video.tmos
plugin.library.node.editor    plugin.video.daserstemediathek
plugin.video.mediathek        plugin.video.rocketbeans
plugin.video.unithek
plugin.video.3satmediathek    plugin.video.fsgo
plugin.video.mlbbasesloaded    plugin.video.shoutfactorytv
plugin.video.viaplay
plugin.video.adverts        plugin.video.funkmediathek
plugin.video.mlbtv        plugin.video.simpsonsworld
plugin.video.videolina
plugin.video.ardmediathek_de    plugin.video.hardstyletv
plugin.video.mmlive        plugin.video.sportschau
plugin.video.wdrmediathek
plugin.video.brmediathek    plugin.video.hrmediathek
plugin.video.ndrmediathek    plugin.video.spurs-tv
plugin.video.wdrrockpalast
plugin.video.catchuptvandmore    plugin.video.iplayerwww
plugin.video.nfl.gamepass    plugin.video.srmediathek
plugin.video.zdf_de_lite


THe file plugin.video.tehnik/ cannot be found under krypton, if I remove it
it won’t match any file.


deltha-MBP-2:repo-plugins user$ git rm -r plugin.video.tehnik/
fatal: pathspec 'plugin.video.tehnik/' did not match any files

Now I will pull my addon into the krypton branch of repo-plugin using
subtree to merge


deltha-MBP-2:repo-plugins user$ git read-tree --prefix=plugin.video.tehnik/
-u myaddon_branch

I am looking for the commited files, there are pages containing the files
from my addon as they have been added.



deltha-MBP-2:repo-plugins user$ git diff --staged
diff --git a/plugin.video.tehnik/INSTALL.txt
b/plugin.video.tehnik/INSTALL.txt
new file mode 100755
index 00000000..c74251be
--- /dev/null
+++ b/plugin.video.tehnik/INSTALL.txt
@@ -0,0 +1,70 @@
+
+INSTALL
+=======
+
+
+EN:
+
+INSTALLING TehniK
+
+Zip archives required:
+
+script.module.xbmcswift2-2.4.0.zip
+plugin.video.tehnik-2.0.7.zip
+
+The TehniK plugin depends on the xbmcswift2 module.
+First you will need install xbmcswift2 and then install TehniK.
+
+Steps:
+
+1. Files will be download locally to the storage device where KODI is
installed.
+2. These files will be installed:
+
+Supplements will select menu -> Install from zip files.
+It will select and install the first file
+script.module.xbmcswift2-2.4.0.zip
+and then
+plugin.video.tehnik-2.0.7.zip
+
+In the bottom right of the screen installer will
+return a message about the completion of the installation
+for each module.
+
+3. Return to the main menu.
+
+From the main menu select video -> addons
+where you will be able to run the new icon "TehniK".
+
+
+RO:
+
+INSTALAREA PLUGINULUI TehniK
+
+Arhive zip necesare:
+
+script.module.xbmcswift2-2.4.0.zip





I am commiting my changes:


deltha-MBP-2:repo-plugins user$ git commit -m "[plugin.video.tehnik] 2.0.7"
[krypton bc7b4b24] [plugin.video.tehnik] 2.0.7
 73 files changed, 924 insertions(+)
 create mode 100755 plugin.video.tehnik/INSTALL.txt
 create mode 100755 plugin.video.tehnik/LICENSE.txt
 create mode 100755 plugin.video.tehnik/addon.py
 create mode 100755 plugin.video.tehnik/addon.xml
 create mode 100755 plugin.video.tehnik/changelog.txt
 create mode 100755 plugin.video.tehnik/fanart.jpg
 create mode 100755 plugin.video.tehnik/icon.png
 create mode 100755
plugin.video.tehnik/resources/language/English/strings.xml
 create mode 100755
plugin.video.tehnik/resources/language/Romanian/strings.xml
 create mode 100755 plugin.video.tehnik/resources/media/allview.png
 create mode 100755 plugin.video.tehnik/resources/media/altex.png
 create mode 100755 plugin.video.tehnik/resources/media/arenait.png
 create mode 100755 plugin.video.tehnik/resources/media/asus.png
 create mode 100755 plugin.video.tehnik/resources/media/bitdefender.png
 create mode 100755 plugin.video.tehnik/resources/media/canonromania.png
 create mode 100755 plugin.video.tehnik/resources/media/celtv.png
 create mode 100755 plugin.video.tehnik/resources/media/creativemonkeyz.png
 create mode 100755 plugin.video.tehnik/resources/media/digi.jpg
 create mode 100755 plugin.video.tehnik/resources/media/domo.png
 create mode 100755 plugin.video.tehnik/resources/media/emag.png
 create mode 100755 plugin.video.tehnik/resources/media/evolio.png
 create mode 100755 plugin.video.tehnik/resources/media/f64.png
 create mode 100755 plugin.video.tehnik/resources/media/flanco.png
 create mode 100755 plugin.video.tehnik/resources/media/fotostart.png
 create mode 100755 plugin.video.tehnik/resources/media/gadget-talk.png
 create mode 100755 plugin.video.tehnik/resources/media/gadgeton.png
 create mode 100755 plugin.video.tehnik/resources/media/gadgetro.png
 create mode 100755 plugin.video.tehnik/resources/media/gamaganda.png
 create mode 100755 plugin.video.tehnik/resources/media/gamextv.png
 create mode 100755 plugin.video.tehnik/resources/media/gaminghd.png
 create mode 100755 plugin.video.tehnik/resources/media/gbuhnici.png
 create mode 100755 plugin.video.tehnik/resources/media/go4it.png
 create mode 100755 plugin.video.tehnik/resources/media/gsmland.png
 create mode 100755 plugin.video.tehnik/resources/media/hp.png
 create mode 100755 plugin.video.tehnik/resources/media/lab501.jpg
 create mode 100755 plugin.video.tehnik/resources/media/lenovo.png
 create mode 100755 plugin.video.tehnik/resources/media/marketonline.png
 create mode 100755 plugin.video.tehnik/resources/media/maxinfinite.png
 create mode 100755 plugin.video.tehnik/resources/media/mediadot.png
 create mode 100755 plugin.video.tehnik/resources/media/mediagalaxy.png
 create mode 100755 plugin.video.tehnik/resources/media/mobiletrend.jpg
 create mode 100755 plugin.video.tehnik/resources/media/mobilissimo.png
 create mode 100755 plugin.video.tehnik/resources/media/nikonisti.png
 create mode 100755 plugin.video.tehnik/resources/media/nokia.png
 create mode 100755 plugin.video.tehnik/resources/media/nowtechro.jpg
 create mode 100755 plugin.video.tehnik/resources/media/orange.jpg
 create mode 100755 plugin.video.tehnik/resources/media/panasonic.png
 create mode 100755 plugin.video.tehnik/resources/media/pc-garage-srl1.jpg
 create mode 100755 plugin.video.tehnik/resources/media/pcgarage-red.png
 create mode 100755 plugin.video.tehnik/resources/media/pgl.png
 create mode 100755 plugin.video.tehnik/resources/media/philips.png
 create mode 100755 plugin.video.tehnik/resources/media/playtech.png
 create mode 100755 plugin.video.tehnik/resources/media/pni.png
 create mode 100755 plugin.video.tehnik/resources/media/prosoft.png
 create mode 100755 plugin.video.tehnik/resources/media/prostore.png
 create mode 100755 plugin.video.tehnik/resources/media/samsung.png
 create mode 100755 plugin.video.tehnik/resources/media/schrack.png
 create mode 100755 plugin.video.tehnik/resources/media/sony.png
 create mode 100755 plugin.video.tehnik/resources/media/telecomtv.jpg
 create mode 100755 plugin.video.tehnik/resources/media/telefonultau.png
 create mode 100755 plugin.video.tehnik/resources/media/telekom.jpg
 create mode 100755 plugin.video.tehnik/resources/media/thephonegeeks.png
 create mode 100755 plugin.video.tehnik/resources/media/tira.png
 create mode 100755 plugin.video.tehnik/resources/media/tiranianu.png
 create mode 100755 plugin.video.tehnik/resources/media/toshiba.jpg
 create mode 100755 plugin.video.tehnik/resources/media/unboxarena.png
 create mode 100755 plugin.video.tehnik/resources/media/upc.jpg
 create mode 100755 plugin.video.tehnik/resources/media/videotutorialro.png
 create mode 100755 plugin.video.tehnik/resources/media/vodafone.jpg
 create mode 100755 plugin.video.tehnik/resources/media/voievozii.png
 create mode 100755 plugin.video.tehnik/resources/media/wararenagaming.png
 create mode 100755 plugin.video.tehnik/resources/media/zonait.png
 create mode 100755 plugin.video.tehnik/resources/settings.xml

Step 2. You wrote that I need to push the changes to my branch:

2. Push the changes to YOUR branch
So origin is now git@github.com:deltha/repo-plugins.git

deltha-MBP-2:repo-plugins user$ git remote -v
myaddon    git@github.com:deltha/plugin.video.tehnik.git (fetch)
myaddon    git@github.com:deltha/plugin.video.tehnik.git (push)
origin    git@github.com:deltha/repo-plugins.git (fetch)
origin    git@github.com:deltha/repo-plugins.git (push)

I am pushing it into krypton

deltha-MBP-2:repo-plugins user$ git push origin krypton
Counting objects: 80, done.
Delta compression using up to 8 threads.
Compressing objects: 100% (72/72), done.
Writing objects: 100% (80/80), 17.73 MiB | 2.39 MiB/s, done.
Total 80 (delta 7), reused 66 (delta 6)
remote: Resolving deltas: 100% (7/7), completed with 1 local object.
To github.com:deltha/repo-plugins.git
   742a0fbe..bc7b4b24  krypton -> krypton


The next step, step 3:

3. Then do the PR here, referencing your branch as source and ours as base.
As BASE I have chosen your Krypton and my krypton branch.
under
https://github.com/deltha/repo-plugins
I pressed the button for a new PR
I have clicked compare across forks
Able to merge. These branches can be automatically merged.

Commits on Apr 17, 2017

￼
deltha
[plugin.video.tehnik] 2.0.7


bc7b4b2

Showing 73 changed files with 924 additions and 0 deletion.




On Mon, Apr 17, 2017 at 1:02 AM, Razzeee <notifications@github.com> wrote:

> @deltha <https://github.com/deltha>
> You should not try to merge our branches against each other.
>
>    1. You need to make your own fork (hit fork here
>    https://github.com/xbmc/repo-plugins
>    <https://github.com/xbmc/repo-plugins>)
>    2. Push the changes to YOUR branch
>    3. Then do the PR here, referencing your branch as source and ours as
>    base.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/xbmc/repo-plugins/pull/1158#issuecomment-294376843>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AHpfJ9xJXK01INrl9vizdypjvJzj5EoNks5rwo_ugaJpZM4M-vRL>
> .
>
